Add schedule to website

RealOrangeOne opened the pull request:
Fixes https://github.com/srobo/tasks/issues/785

WillB97 commented on _events/sr2022/competition.md:
```diff
@@ -12,6 +12,32 @@ The Student Robotics Competition marks the ultimate stage of SR2022, showcasing
+13:30 | Competition Photo
+14:00 | Knockouts
+16:20 | Prize Ceremony
+16:45 | Kit Return
```
The end-of-day on Sunday is just a nominal time. In practice this whenever they've left by.

RealOrangeOne replied:
We should still advertise a time as a "Please be gone by this time"

PeterJCLaw replied:
On Sunday evening people may have quite a way to travel. If we're advertising an End time then they'll likely assume that's a time they can actually be leaving at, so us having a bottleneck at the exit which makes that time less realistic doesn't seem fair.

Back to my original question: is 15 minutes enough time for Kit Return?

I don't know what's involved in the return, however often it involves someone looking in their kit to ensure that everything is actually present. This takes a while per team (certainly > 2 minutes) and while it can parallelise very well desk space quickly becomes the limiting factor.

To this end, many teams often start deconstructing their robots as soon as they're knocked out of the knockouts, in part so they can get away quickly at the end of the day. We typically therefore want to have kit return actually available throughout the afternoon to facilitate this (even if we don't actually advertise it).

Has this all been considered in the allowance of only 15 minutes for Kit Return?

WillB97 replied:
Can I suggest this is discussed on the issuea that decides the schedule (https://github.com/srobo/tasks/issues/743) and kit handback (https://github.com/srobo/tasks/issues/759).

PeterJCLaw replied:
-> https://github.com/srobo/tasks/issues/759#issuecomment-1086371945

PeterJCLaw replied:
The conclusion in a recent Competition Team meeting was that we should ship this as-is and we can amend it if needed.

PeterJCLaw commented on _events/sr2022/competition.md:
```diff
@@ -10,7 +10,35 @@ The Student Robotics Competition marks the ultimate stage of SR2022, showcasing
 ## Schedule
 
-The competition will be held from *Saturday 23rd April* until *Sunday 24th April*. Doors open 9am and we aim to finish around 6pm.
+The competition will be held from *Saturday 23rd April* until *Sunday 24th April*. Doors open 9am and we aim to finish around 5pm.
```
nit: doors open 8:45 on Sunday. I'd suggest drop the second sentence here (and maybe even the whole paragraph) now that the details are in the tables below.

WillB97 commented:
Is there a reason we've made the map lower resolution and removed the nearby motorway labelling.

RealOrangeOne commented:
> Is there a reason we've made the map lower resolution and removed the nearby motorway labelling.

Sort of. It was the best cropping I could do. I removed the motorway labelling as it's not really correct.

PeterJCLaw commented on _events/sr2022/competition.md:
```diff
@@ -10,7 +10,35 @@ The Student Robotics Competition marks the ultimate stage of SR2022, showcasing
+
+Time | Activity
+--- | ---
+8:45 | Doors Open
```
```suggestion
9:00 | Doors Open
```

Scarzy commented on _events/sr2022/competition.md:
```diff
@@ -10,7 +10,35 @@ The Student Robotics Competition marks the ultimate stage of SR2022, showcasing
+16:45 | Kit Return
+17:00 | End of Day
+
+On day 2, a marathon event is being held in Southampton. For parking, please approach the campus from the west, and arrive as prompt as possible. We also strongly recommend bringing a lunch, as it will be difficult to get to the local shops.
```
Add a note to keep an eye on this page for up to date information
